.github/workflows/auto_catchup.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         GITHUB_TOKEN: ${{secrets.PAT}}
         SOURCE_BRANCH: "master"
         SOURCE_REPO: "RocketChat"
-        TARGET_BRANCH: "develop_pwa"
+        TARGET_BRANCH: "develop"
```

Comment:
I think we should disable this workflow until we have gotten all the way up to 5.x.  Can you comment out lines 4 and 5 please?  thanks.